Signin token expired

```
Error:  ExecError: ExecError: (1) Generating ephemeral keys...
Retrieving signed certificate...
Non-interactive mode detected, using device flow.
Error: signing [renovate/renovate:31.13]: getting signer: getting key from Fulcio: retrieving cert: error obtaining token: expired_token
main.go:46: error during command execution: signing [renovate/renovate:31.13]: getting signer: getting key from Fulcio: retrieving cert: error obtaining token: expired_token
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -4,6 +4,10 @@ on:
   push:
   pull_request:
 
+permissions:
+  contents: read
+  id-token: write
+
 jobs:
   build:
     runs-on: ubuntu-latest

--- a/.github/workflows/cron.yml
+++ b/.github/workflows/cron.yml
@@ -7,6 +7,10 @@ on:
   schedule:
     - cron: '0 1 * * *'
 
+permissions:
+  contents: read
+  id-token: write
+
 jobs:
   build:
     runs-on: ubuntu-latest

